flow/ondemand/client/client.go

```diff
@@ -378,6 +378,21 @@ func (o *OnDemandProbeClient) Stop() {
 	o.checkForRegistration.Stop()
 }
 
+// Invoke capture upon start of analyzer on selected from config SubGraph
+func (o *OnDemandProbeClient) InvokeCaptureFromConfig() () {
+	capture_gremlin := config.GetConfig().GetString("analyzer.startup_capture")
+	capture_bpf := config.GetConfig().GetString("analyzer.startup_capture_bpf")
+	if(capture_gremlin == "" && capture_bpf == "") {
+		return
+	}
+	logging.GetLogger().Infof("Try to invoke capturing from the startup with gremlin: %s and BPF: %s", capture_gremlin, capture_bpf)
+	capture := types.NewCapture(capture_gremlin, capture_bpf)
+	capture.SocketInfo = true
+	capture.Type = "pcap"
```

Comment:
Again here, would we not want to expose `Type` configuration also?

Reply:
ok

Reply:
again, we prefer to keep less parameters.